ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/TransactionPoolFactory.java

```diff
@@ -33,7 +33,12 @@ public static TransactionPool createTransactionPool(
       final Clock clock,
       final int maxPendingTransactions,
       final MetricsSystem metricsSystem,
-      final SyncState syncState) {
+      final SyncState syncState,
+      final int pendingTransactionRetentionPeriod) {
```

Comment:
This argument needs to be grouped next to maxPendingTransactions.  between line 34 and 35